### DISCUSSION TITLE
Access to elementaryOS 6 Builds

### DISCUSSION THREAD

**Discussion by zagatta** [elementary/os#441] [Q&A] [answered]
I contributed with 25$ to the [App Center for Everyone campaign on indiegogo](https://www.indiegogo.com/projects/appcenter-for-everyone/x/23078319#/).

Unfortunately I have not found out how to get an early-access build for elementaryOS 6. I have gotten in contact via indiegogo but gotten no response there.

Since github is the new designated platform for asking questions, I'll try my luck here.

**Comment by danirabbit** [comment, marked as answer]
Hey Jonas, 

Thanks again for backing us on IndieGoGo! I replied to your email a little over a week ago but I guess it must not have gone through. I've added your username to the allow list on https://builds.elementary.io. You should have login access very soon. Enjoy :)

**Reply by zagatta** [reply]
Awesome!
